.github/bin/minikube/test-olm.sh

```diff
@@ -25,12 +25,14 @@ trap "catchFinish" EXIT SIGINT
 
 runTest() {
   local channel=next
+  local catalogImage=quay.io/eclipse/eclipse-che-kubernetes-opm-catalog:next
```

Comment:
I think for ci we have to use ${IMAGE_REGISTRY_HOST} instead of `quay.io` to build and push catalog images to the private docker registry enabled on the minikube and use them in the  test... That's a how tests are working currently.

Reply:
Fixed